frontend/src/components/GrantAccess.vue

```diff
@@ -210,22 +241,56 @@ function statusSelected(evt: any) {
                         placeholder="Forest Client Id - 8 digits"
                         v-model="formData.forestClientNumber"
                         v-on:keypress="onlyDigit($event)"
+                        v-bind:disabled="loading"
+                        @input="forestClientNumberChange()"
                         :validateOnChange="true"
-                        :class="{ 'is-invalid': errors.forestClientNumber }"
+                        :class="{
+                            'is-invalid': errors.forestClientNumber,
+                        }"
                     />
                     <ErrorMessage
                         class="invalid-feedback"
                         name="forestClientNumber"
                     />
                 </div>
+                <div class="col-md-3 d-flex align-items-end">
+                    <button
```

Comment:
I just wondering if we want to switch to primevue outlined button, cause otherwise we have to change it in the future anyway